mocha.js

```diff
@@ -13074,7 +13074,7 @@ var process = module.exports = {};
 var cachedSetTimeout;
 var cachedClearTimeout;
 
-function defaultSetTimout() {
+function defaultSetTimeout() {
```

Comment:
To answer a concern from the description: given that this function seems to be local, I don't think it's an API, let alone a public one, so as long as all the references to it were updated this should be fine. (Now, if the name of something it is assigned to were updated, we'd need to check whether *that* is an API, but I'm pretty sure none of the left-sides for this were changed...)

Reply:
Ok, turns out this is coming from: https://github.com/defunctzombie/node-process/blob/master/browser.js (or some variant of that). If you can help find the right project for it, that'd help me.

Reply:
I'd imagine that that is in fact the right project. As I understand it, this defunctzombie guy maintains a bunch of packages that are basically clones of the Node builtins that can be pulled in by Browserify and Webpack.